csharp/ql/test/library-tests/dataflow/flowsources/local/environment/EnvironmentVariableFlow.ql

```diff
@@ -0,0 +1,16 @@
+import csharp
+import semmle.code.csharp.dataflow.internal.ExternalFlow
+
+module EnvironmentVariableFlowConfig implements DataFlow::ConfigSig {
+  predicate isSource(DataFlow::Node source) { sourceNode(source, "environment") }
```

Comment:
nit: You can consider to do this using threat models (also fine to leave it as it is).